### PR TITLE
Storybook v7

### PR DESCRIPTION
**Pull request by tackc**
#### Please check if the PR fulfills these requirements

- [x] The changes don't come from your master branch. ([Source](https://blog.jasonmeridth.com/posts/do-not-issue-pull-requests-from-your-master-branch/)) 

---

#### What kind of change does this PR introduce? (bug fix, feature, docs update, ...)
This PR updates all necessary components to work with Storybook 7.
I also resolved numerous outstanding lint errors that were occurring 

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] This change requires a documentation update

#### What is the current behavior? (You can also link to an open issue here)
Storybook v7 had numerous breaking changes. 

#### Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)
I have not discovered any breaking changes related to these updates

### CODE REVIEW

File: src/Services/AuthWrapper.js [hunk 1]
```diff
@@ -1,34 +1,29 @@
-/* eslint-disable react/jsx-props-no-spreading */
-import React, { useEffect } from 'react';
-import { useNavigate, useParams, useLocation } from 'react-router-dom';
+import { useEffect } from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
 
-const isAuthenticated = () => {
+export const isAuthenticated = () => {
```
Comment (renburnett) [inline src/Services/AuthWrapper.js:4]: @tackc do we need to export this? I think the AuthWrapper should be the only thing we need to export

**Review by renburnett — APPROVED** [review]
@tackc pulled it down and played with it-- looks good to me. the only thing I noticed was that the `BreadCrumb` component doesnt seem to be rendering in storybook. I assume it's unrelated? also made a comment about the export in authwrapper. all small potatoes-- feel free to merge after you tweak (or if i'm wrong with my comment, feel free to merge without changing)

File: src/Services/AuthWrapper.js (hunk 1 above)
Comment (tackc) [reply src/Services/AuthWrapper.js:4]: It's used on `ErrorPage` to conditionally render the <Navbar /> component. I tried removing it and the app didn't compile

**Comment by tackc** [comment]
I also fixed the breadcrumb story. It didn't have any sample data in there before 🤷‍♂️

**Pull request merged**